docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+
+version: "3"
+services:
+    postgres:
+      image: postgres
```

Comment:
This would get postgresql v13 - I haven't tried it (as the host where I run the site doesn't offer it) but I assume it works?